src/components/dashboard/Tasks/hooks/useTasks.js

```diff
@@ -0,0 +1,26 @@
+//@flow
+import React from 'react'
+import { t } from '@lingui/macro'
+
+import { useInviteCopy, useInviteShare } from '../../../invite/useInvites'
+import { ShareInviteButton } from '../../../invite/Invite'
+import { isSharingAvailable } from '../../../../lib/share'
+
+export const useTaskList = () => {
+  const { copy: inviteCopy } = useInviteCopy()
+  const { share } = useInviteShare()
+  const actionText = isSharingAvailable ? t`Share invite Link` : t`Copy invite link`
+
+  // todo: how/where to define current / active task(s)?
```

Comment:
For just an mvp I don't think it is necessary to be in scope of this pr, can be new issue
but if we have plans to add new tasks later on
I think we need some setup so that it can be managed also outside of code

@johnsmith-gooddollar @sirpy
you have any suggestions ?

Reply:
as discussed with @johnsmith-gooddollar not part of this pr, created separate issue for.
need to sync with @patpedrosa as well